metabase/metabase
# Show xrays on the new homepage

Epic: https://github.com/metabase/metabase/issues/21322

How to test:
- Run a fresh Metabase instance. You should see x-rays for the sample database.
- Add another database. You can copy the connection string from the sample database. You should see x-rays for that database and a link to browse it.
- Ignore broken e2e tests for now

<img width="920" alt="Screenshot 2022-03-31 at 20 12 48" src="https://user-images.githubusercontent.com/8542534/161113734-cb6dff13-e70f-4c71-a52e-ee3ba1d287a5.png">
<img width="792" alt="Screenshot 2022-03-31 at 20 13 48" src="https://user-images.githubusercontent.com/8542534/161113756-19cdbd33-500a-495d-b11f-174667e539bd.png">
